control-plane/roles/logging/tasks/main.yaml

```diff
@@ -0,0 +1,44 @@
+---
+- name: Gather release versions
+  setup_yaml:
+
+- name: Create namespace {{ logging_namespace }}
+  k8s:
+    definition:
+      apiVersion: v1
+      kind: Namespace
+      metadata:
+        name: "{{ logging_namespace }}"
+        labels:
+          name: "{{ logging_namespace }}"
+
+- name: Deploy GCP secret for loki
+  k8s:
+    namespace: "{{ logging_namespace }}"
+    definition:
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: "{{ logging_namespace }}"
+      data:
+        gcs.json: "{{ logging_gcp_service_account_json }}"
+  when: logging_gcp_service_account_json is not none
+
+# Stores logs
+- name: Deploy loki-stack
+  include_role:
+    name: ansible-common/roles/helm-chart
+  vars:
+    helm_repo: "https://grafana.github.io/helm-charts"
```

Comment:
Can be used from release vector.

Reply:
https://github.com/metal-stack/metal-roles/commit/0cd1d19733b13dfd2b53bd8721744471f08ff1c3